web/packages/teleport/package.json

```diff
@@ -39,7 +39,8 @@
     "@xterm/addon-webgl": "^0.18.0",
     "@xterm/xterm": "^5.5.0",
     "create-react-class": "^15.6.3",
-    "events": "3.3.0"
+    "events": "3.3.0",
+    "react-markdown": "^9.0.3"
```

Comment:
I haven't seen it brought up in this PR, but [react-markdown weighs 44.5 kB minified and gzipped](https://bundlephobia.com/package/react-markdown@9.0.3) which significantly increased the size of the bundle:

```
before 406f808: webassets/teleport/app/app.js  3,174.86 kB │ gzip:   893.18 kB
after  406f808: webassets/teleport/app/app.js  3,301.29 kB │ gzip:   931.84 kB
delta:                                          +126.43 kB │ gzip:   +38.66 kB
```